### PR TITLE
Fallback to local file when Fedora object isn't 200 OK

### PR DESCRIPTION
**Pull request by tonyc**
Per email discussion with Mike K:

update the production codepath for authority file importing to fall back to local cached copy if the remote file in fedora doesn't return HTTP 200 OK

ping @val99erie for review

### CODE REVIEW

File: config/initializers/state_authorities.rb [hunk 1]
```diff
@@ -1,20 +1,28 @@
 # Import the state authority data
 
-uri_for_state_file = 'http://dl.tufts.edu/file_assets/generic/tufts:state-authority/0'
+require 'net/http'
+uri_for_state_file = URI('http://dl.tufts.edu/file_assets/generic/tufts:state-authorit-1213y/0')
```
Comment (val99erie) [inline config/initializers/state_authorities.rb]: This URI got mangled somehow.  It should be:
http://dl.tufts.edu/file_assets/generic/tufts:state-authority/0

Comment (tonyc) [inline config/initializers/state_authorities.rb]: Thank you! Good catch. Code reviews ftw.

Comment (tonyc) [inline config/initializers/state_authorities.rb]: Fix pushed.

File: config/initializers/state_authorities.rb [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
   puts "Downloading state authority from #{uri_for_state_file}"
 
-  File.open(filename, 'w') do |file|
-    open uri_for_state_file do |f|
-      f.each_line { |line| file.write(line.force_encoding('UTF-8')) }
+  response = Net::HTTP.get_response(uri_for_state_file)
+
+  if response.code == '200'
+    File.open(filename, 'w') do |file|
+      file.write response.body
```
Comment (val99erie) [inline config/initializers/state_authorities.rb]: I think we need something like this:
file.write response.body.force_encoding('UTF-8')

Else the download fails if you remove the cached file.
Should we change the other 3 files to force encoding too?

**Pull request merged**